test/acm/bike-rental-app/kustomization.yaml

```diff
@@ -4,22 +4,62 @@ kind: Kustomization
 resources:
 - ../../../acm/registration/near-edge/overlays/bike-rental-app/
 
-namespace: my-test-namespace
+namePrefix: custom-prefix-
 
 patches:
 - patch: |-
     - op: replace
-      path: /spec/channel
-      value: my-test-namespace/bike-rental-app-channel
+      path: /spec/template/metadata/name
+      value: '{{name}}-grdryn-bike-rental-app'
```

Comment:
I assume your username was not intended to be included?

Reply:
Correct, good catch, thanks! I've fixed it now.

Related to this, I'm not sure how useful these test/dev kustomizations are now, given that we don't all share a single cluster. Maybe once this PR is merged, I'll propose removing them in a separate PR, to see if there are any objections or if there's anybody still using them.